src/Orckestra.Composer.Website/App_Code/OrdersHelper.cshtml

```diff
@@ -23,6 +23,13 @@
                                     @Html.Localize("MyAccount", "B_CancelEditing")
                                 </a>
                             </li>
+                        <li v-if="order.OrderInfos.IsOrderEditable &amp;&amp; order.OrderInfos.IsOrderCancelable &amp;&amp; !order.OrderInfos.IsBeingEdited">
+                                <a v-bind:data-orderid="order.OrderInfos.OrderNumber" data-qa="cancel-order" data-oc-click="cancelOrderConfirm" class="cta" href="#">
+                                    <span>@Html.Localize("General", "B_CancelOrder")</span>
+                                </a>
+
+                                @cancelOrderConfirmationModal("cancelOrderModal", "{{order.OrderInfos.OrderNumber}}")
```

Comment:
you need render this modal just once - not for each item
also move data-oc-click="cancelOrderConfirm" to Vue and have sparameter orderNumber
v-on:click.prevent="cancelOrderConfirm(order.OrderInfos.OrderNumber)"
in vue have property OrderNumber and use it inside modal - to setup active OrderNumber